modules/apps/object/object-web/src/main/resources/META-INF/resources/list_type_definitions/configuration/icon/import_list_type_definition.jsp

```diff
@@ -8,22 +8,13 @@
 <%@ include file="/init.jsp" %>
 
 <div>
-
-	<%
-	Map<String, Object> warningModalText = HashMapBuilder.<String, Object>put(
-		"body", Arrays.asList(LanguageUtil.format(request, "there-is-another-x-with-the-same-external-reference-code-as-the-imported-one", StringUtil.toLowerCase(LanguageUtil.get(request, "picklist"))), LanguageUtil.format(request, "before-importing-the-new-x-you-may-want-to-back-up-its-entries-to-prevent-data-loss", StringUtil.toLowerCase(LanguageUtil.get(request, "picklist"))), LanguageUtil.get(request, "do-you-want-to-proceed-with-the-import-process"))
-	).put(
-		"header", LanguageUtil.get(request, "update-existing-picklist")
-	).build();
-	%>
-
```

Comment:
This commit should be squashed with the one that you introduce this change.
Since the reviewer can review the PR per commit, a change in the same class but in different commits can slow down the review process because the reviewer will stop to write a comment and this suggestion is implemented in X commits latter.

Reply:
If you don't want to refactor it now, you can leave it as it is.
However, I would like you to keep this tip for the future PRs.

Reply:
Hey @marcelabc, I will make this change now, thanks for the feedback!